components/admin_console/system_users/list/system_users_list.tsx

```diff
@@ -331,6 +331,7 @@ export default class SystemUsersList extends React.PureComponent<Props, State> {
         return (
             <div>
                 <SearchableUserList
+                    actionUserProps={{}}
```

Comment:
this is similar to https://github.com/mattermost/mattermost-webapp/pull/11328#discussion_r996916789